src/main/java/uk/gov/hmcts/cmc/claimstore/services/legaladvisor/DocAssemblyApi.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.cmc.claimstore.services.legaladvisor;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@FeignClient(name = "doc-assembly-api", url = "${docAssembly.api.url}")
+public interface DocAssemblyApi {
```

Comment:
If we don't already have the client repository for doc assembly, we should think to extract this in a reusable, like we have CCD client and DM client.

Reply:
@dharmendrak also agrees. Doing it 👍